### ISSUE TITLE
NVDA does not read the state of the combo box(as Read only) 

### ISSUE THREAD

**Issue by yugimamidi** [nvaccess/nvda#8763]
#### Have you tried any other versions of NVDA?: `No

[bug-1222636_ext-html.zip](https://github.com/nvaccess/nvda/files/2395669/bug-1222636_ext-html.zip)

`

**Comment by leonardder** [comment]
You can also open the attached example like this from your address bar:

```
data:text/html, <select role="combobox" title="readonly" aria-readonly="true"><option value="volvo">Volvo</option><option value="saab">Saab</option><option value="opel">Opel</option><option value="audi">Audi</option></select>
```

**Comment by Adriani90** [comment]
This is only reproducible in Edge Edgehtml. Since this will be more than likely solved in Edge chromium (it works properly in Google Chrome 79), I am closing this issue.
Tested with NVDA 2019.3 Beta 1.
